airflow/contrib/kubernetes/pod.py

```diff
@@ -250,8 +250,8 @@ def _extract_ports(ports):
 
 def _extract_resources(resources):
     if isinstance(resources, k8s.V1ResourceRequirements):
-        requests = resources.requests
-        limits = resources.limits
+        requests = resources.requests or {}
+        limits = resources.limits or {}
```

Comment:
You somehow got some extra unrelated changes committted.

Reply:
fixed